requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with python 3.6
 # To update, run:
 #
-#    pip-compile --generate-hashes --output-file=requirements.txt
+#    pip-compile --generate-hashes
```

Comment:
This is helpful for people who never used pip-compile - could you please run `pip-compile --generate-hashes --output-file=requirements.txt` so we have the text `--output-file=requirements.txt` back here.

Reply:
Sure!

Reply:
I've regenerated it and tested with `tox`, everything ok. 

I'm sending the new commit.